dspace-api/src/test/java/org/dspace/content/CollectionTest.java

```diff
@@ -180,6 +191,10 @@ public void testCreateWithValidHandle() throws Exception {
         // check that collection was created, and that its handle was set to proper value
         assertThat("testCreateWithValidHandle 0", created, notNullValue());
         assertThat("testCreateWithValidHandle 1", created.getHandle(), equalTo("987654321/100"));
+
+        } finally {
+            configurationService.setProperty("handle.additional.prefixes", handleAdditionalPrefixes);
```

Comment:
You can remove this logic, as there's no need to reset Configuration values.  They automatically reset after every test.  So, the try/finally can be removed & the `getProperty` at the top.   Just call `setProperty` to change the value, and it'll reset automatically.

Reply:
At a first glance, I haven't found anything like a reset configuration. This maybe happens only for IntegrationTests?

Reply:
@alemarte : My mistake. You are correct, it seems this configuration reload is only currently enabled for Integration Tests here: https://github.com/DSpace/DSpace/blob/main/dspace-api/src/test/java/org/dspace/AbstractIntegrationTestWithDatabase.java#L192-L193

So, you are correct that this code is unfortunately necessary until we add similar logic to the AbstractUnitTest.destroy() method here: https://github.com/DSpace/DSpace/blob/main/dspace-api/src/test/java/org/dspace/AbstractUnitTest.java#L145

If you are willing, we could add that code to `AbstractUnitTest.destroy()` now.  However, if you'd rather keep this PR as-is, I'm OK with this moving forward and we can do a future refactor to remove this logic once `AbstractUnitTest` properly resets configurations to their defaults.